src/server/bg_services/agent_blocks_reclaimer.js

```diff
@@ -109,8 +112,8 @@ class AgentBlocksReclaimer {
     /**
      * @override in unit tests for decoupling dependencies
      */
-    populate_nodes_for_blocks(...args) {
-        return MDStore.instance().populate_nodes_for_blocks(...args);
+    populate_nodes_for_blocks(blocks) {
+        return map_server.prepare_blocks_from_db(blocks);
```

Comment:
Notice that now we populate all of the block (frags, chunks, nodes).
Before we've only populated the nodes since we do not need any of the rest.
It shouldn't affect the code since it will populate the nodes as well but just need to make sure.

Reply:
true. a price need to be paid when working with new BlockDB and BlockAPI classes